recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+    
+
```

Comment:
Can you remove the extra line.  (That is just style, but these recipes will be public and handled by the community, so style is important to keep the recipes clean :wink:)